modules/swagger-parser-v3/src/test/java/io/swagger/parser/test/OpenAPIV3ParserTest.java

```diff
@@ -183,6 +184,12 @@ public void test30NoOptions(@Injectable final List<AuthorizationValue> auths) th
         Assert.assertEquals(result.getOpenAPI().getComponents().getSchemas().get("OrderRef").getType(),"object");
     }
 
+    @Test
+    public void testShellMethod(){
+        OpenAPI openAPI = new OpenAPIV3Parser().read("https://gist.githubusercontent.com/webron/e3b0650dfcc06fe8236841fe599c287f/raw/12512eb5343dd56ce79369d7ff58072584bd0dc7/openapi.yaml");
```

Comment:
It's not so good to have remote calls on tests, since the idea is to execute them in local enviroments. I suggest to mock this call. So you can evaluate the behavior you expect without making remote calls.